setup.md

```diff
@@ -2,29 +2,94 @@
-[their website](https://sqlitebrowser.org/dl/.)
+# Overview
 
-If you are running **Linux**, you may already have SQLite3 installed, please use the command 
-`which sqlite3` to see the path of the program, otherwise you should be able to get it 
-from your package manager (on Debian/Ubuntu, you can use the command `apt install sqlite3`).
+To participate in a Software Carpentry workshop, you will need access to software as described below. In addition, you will need an up-to-date web browser.
 
-If you are running **Windows**, run installers as administrator.
-Additionally, make sure you select the right installer version for your system.
-We recommend that you use [git for Windows](https://gitforwindows.org/).
-This is described in the [UNIX Shell lesson](http://swcarpentry.github.io/shell-novice/setup.html).
-If the installer asks to add the path to the environment variables, check yes, otherwise you have to manually add the path of the executable to the `PATH` environmental variables.
-This path informs the system where to find the executable program.
+We maintain a list of common issues that occur during installation as a reference for instructors that may be useful on the
+[Configuration Problems and Solutions wiki page](https://github.com/swcarpentry/workshop-template/wiki/Configuration-Problems-and-Solutions).
 
-If installing SQLite3 using Anaconda, refer to the [anaconda sqlite docs](https://anaconda.org/anaconda/sqlite).
+# The Bash Shell
 
-After the installation and the setting of the paths, close the terminal and reopen a new terminal.
-This enables paths and configurations to be loaded.
+Bash is a commonly-used shell that gives you the power to do tasks more quickly.
 
-# Files
-Please download the database we'll be using: [survey.db]({{ page.root }}/files/survey.db)
+## Linux
+
+The default shell is usually Bash and there is usually no need to install anything.
+
+To see if your default shell is Bash type `echo $SHELL` in a terminal and press the **Enter** key.
+If the message printed does not end with '/bash' then your default is something else and you can run Bash by typing `bash`.
+
+## MacOS
+
+The default shell in some versions of MacOS is Bash, and Bash is available in all versions, so no need to install anything.
+You access Bash from the Terminal (found in `/Applications/Utilities`). See the Git installation [video tutorial](https://youtu.be/9LQhwETCdwY)
+for an example on how to open the Terminal. You may want to keep Terminal in your dock for this workshop.
+
+To see if your default shell is Bash type `echo $SHELL` in Terminal and press the **Return** key. If the message printed does not end with '/bash'
+then your default is something else and you can run Bash by typing `bash`.
+
+If you want to change your default shell, see this [Apple Support article](https://support.apple.com/en-au/HT208050) and follow the instructions on
+"How to change your default shell".
```

Comment:
Don't know that we need to get into this level of detail for this workshop as work in the shell is limited and we aren't working with shell scripts. The default has been `zsh` since 2019/macOS Catalina. Something shorter?

"The default shell in older versions of macOS was `bash`. The default shell since 2019 is `zsh`, though `bash` is still available. For the purposes of this workshop, the shells are similar enough that either is appropriate."

Reply:
Good suggestion, which I've accepted. I'm going to merge this now.